docs/src/testing/debugging.md

```diff
@@ -13,7 +13,7 @@ Starknet Foundry standard library adds a utility method for printing inside test
 
 ## In tests
 
-Here's a test with example use of print method:
+Here's a test with example use of [`print`](../appendix/forge-library/print.md) method:
 
 ```rust
 // Make sure to import Starknet Foundry PrintTrait
```

Comment:
I think the import below is incorrect, rn it should be `use snforge_std::io::PrintTrait` I believe

Reply:
PrintTrait is reexported in lib.cairo so it should be fine. We show it like this in couple other places.